GUI/QtQuick/qml/Send/AdvancedDetails.qml

```diff
@@ -0,0 +1,501 @@
+import QtQuick 2.12
+import QtQuick.Window 2.12
+import QtQuick.Controls 2.12
+import QtQuick.Layouts 1.15
+
+import "../BsStyles"
+import "../StyledControls"
+
+import wallet.balance 1.0
+
+ColumnLayout  {
+
+    id: layout
+
+    signal sig_continue(signature: var)
+    signal sig_simple()
+    signal sig_select_inputs()
+
+    height: 748
+    width: 1132
+    spacing: 0
+
+    property var tempRequest: null
+
+    RowLayout {
+
+        Layout.fillWidth: true
+        Layout.preferredHeight : 34
+
+        Button {
+            id: simple_but
+
+            Layout.leftMargin: 24
+            Layout.alignment: Qt.AlignLeft | Qt.AlingVCenter
+
+            activeFocusOnTab: true
+
+            font.pixelSize: 13
+            font.family: "Roboto"
+            font.weight: Font.Normal
+            palette.buttonText: "#7A88B0"
```

Comment:
hard-coded colors everywhere - is this intended?

Reply:
Hurried...will fix

Reply:
Very many places to fix. (not only in pr, but in whole code) Start do it, but I think it must be additional task

Reply:
Ok, let's fix only for the scope of this PR, don't touch every other color in QML